src/mame/swtpc/swtpc09.cpp

```diff
@@ -68,23 +68,90 @@
  F100 - ????  CDS disk 0 controller
  F300 - ????  CDS disk 1 controller
 
-The supplied flex and uniflex floppies all worked when this driver was
-created (2014-03-25), but have been broken since sometime in 2015.
-
-The message is the same for all 3 disks: Unable to identify the image format.
-
-Official notes:
-mess swtpc09 -flop1 DMAF2_BOOT.dsk
-press D to boot flex operating system. dir 0 will perform a directory of drive 0.
+** Notes on the use with common disk images **
+
+MAME emulates the floppy disk controller more thoroughly than some other
+common emulators for these systems. Other common emulators can ignore the disk
+density and disk side driven by the software. FLEX numbers sectors
+contiguously from one side to the other so a simple emulator need only map the
+track and sector numbers. Real disk images often mix single and double density
+formats, e.g. single density for the first track, and the software may not
+have been updated to correctly drive the side and density outputs, so disks
+for these other emulators might not not work in this MAME emulator. These disk
+images and their softare need to be updated - this is not working around
+limitations of this MAME emulator, these would not be expected to work on real
+hardware either.
+
+However the disk controllers used by this emulator have some options that can
+help with this transition, help get these disk images running as-is. The disk
+density selection can be overwitten. The disk side selection can be overridden
```

Comment:
Do you mean “density selection can be overridden” or do you actually mean something in the image can be overwritten?

Reply:
The option overrides the drive density selection line, it does not modify the image format. This option it not an impractical hardware modification as a pure double density format disk could be created and the drive density wired for double density, without the boot ROM or OS disk drive on the disk having been updated (might be programmed for single density track 0 etc).

Reply:
Then correct the comment – don’t use “overwritten” if you mean “overridden”.

Reply:
Done.